ably-ios/ARTAuth.m

```diff
@@ -84,8 +86,26 @@ - (ARTAuthOptions *)mergeOptions:(ARTAuthOptions *)customOptions {
     return customOptions ? [self.options mergeWith:customOptions] : self.options;
 }
 
+- (void)storeOptions:(ARTAuthOptions *)customOptions {
```

Comment:
If ARTAuthOptions some day adds new properties, I think chances are this method will be left outdated. Maybe add this functionality to ARTAuthOptions, as described [here](https://developer.apple.com/library/ios/documentation/General/Conceptual/DevPedia-CocoaCore/ObjectCopying.html)?

Why do we need to copy the options anyway?